src/Sarif.Converters/StaticDriverVerifierConverter.cs

```diff
@@ -228,14 +265,25 @@ private void ProcessLine(string logFileLine, Result result)
             }
         }
 
-        private string BuildMessageFromState(string state)
+        private bool IsHarnessOrRulesFiles(string fileName)
         {
-            if (string.IsNullOrEmpty(state) || state == "^")
+            return fileName.EndsWith(".slic") || fileName.EndsWith("sdv-harness.c");
```

Comment:
I use the internal SDV logic for this. I will change it in both places (SDV and converter) in my next change.